### PR TITLE
notaryv2: fix 'notation list'

### PR DESCRIPTION
**Pull request by rchincha**
Signed-off-by: Ramkumar Chinchani <rchincha@cisco.com>

**What type of PR is this?**

'notation list' doesn't return digests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/config-minimal.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
         "ReadOnly": false
     },
     "log": {
-        "level": "debug",
-        "output": "/tmp/zot.log"
```
Comment (hallyn) [inline examples/config-minimal.json:13]: Was this intentional?  It seems unrelated.

Comment (rchincha) [reply examples/config-minimal.json]: yes, removed now.

**Review by hallyn — APPROVED** [review]
